src/css/cart.css

```diff
@@ -69,4 +72,51 @@
     border-color: #5e6a71;
     box-shadow: 0 6px 12px rgba(0, 0, 0, .2);
   }
-  
\ No newline at end of file
+
+/* fake link css for courses to match mcmaster styling for <a>*/
+.fake-link {
+  color: #7d0040;
+/*  text-decoration: underline;*/
+  cursor: pointer;
+  }
+    
+div.fake-link:hover {
+  color: #222;
+  }
+
+/* Could add a large right margin to the first course-link element
+   so the bottom border doesn't touch the cart number bubble overflow-x: */
+.course-link {
+  width: %100;
+  font-size: 20px;
+  padding: 5px 5px 10px 5px;
+  margin: 5px 5px 15px 5px;
+  border-bottom: 1px solid #d4d4d4;
+  display: flex;
+ }
+
+.course-text {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+ }
+
+.cart-minus {

```

Comment:
I _still_ think we should be using glyphicons for a garbage bin/cart/plus-sign. 
There's a trello ticket open for it I think. This is a great first step forward though.